content/authors/aaron-meyers/_index.md

```diff
@@ -4,14 +4,12 @@ first_name: Aaron
 last_name: Meyers
 # List your pronoun(s) if you want them displayed alongside your name.
 # If blank, we'll use just your name. Learn more http://mypronouns.org
```

Comment:
```suggestion
```

Reply:
In some of the author pages, this information is still included. In others, it's not. So I'm highlighting one example here, but I've seen some others as well. 

I think we should remove all prompts (not just the pronouns ones) consistently, especially since this reads as "instructions for the author," and we don't really need to maintain those instructions in the metadata.